pkg/host/store/store.go

```diff
@@ -1,4 +1,4 @@
-package host
+package store
```

Comment:
nit: either put the keyworkd "store" in the package name or in the interface name.

Reply:
renamed store.StoreManagerInterface > store.ManagerInterface